docs/newbs_getting_started.md

```diff
@@ -66,9 +66,14 @@ After Homebrew is installed run these commands:
 
 You will need to install Git and Python. It's very likely that you already have both, but if not, one of the following commands should install them:
 
-* Debian / Ubuntu / Devuan: `apt-get install git python3 && python3 -m pip install qmk`
-* Fedora / Red Hat / CentOS: `yum install git python3 && python3 -m pip install qmk`
-* Arch: `yay -S qmk` (or use any other AUR Helper)
+* Debian / Ubuntu / Devuan: `sudo apt install git python3 python3-pip`
+* Fedora / Red Hat / CentOS: `sudo yum install git python3 python3-pip`
+* Arch / Manjaro: `sudo pacman -S git python python-pip python-setuptools libffi`
+
+
+Install the global CLI to bootstrap your system:
+
+  `python3 -m pip install --user qmk` (on Arch-based distros you can also try the `qmk` package from AUR (**note**: it's maintained by a community member): `yay -S qmk`)
```

Comment:
I'm torn on how we handle arch here. It's a better user experience if they simply install `qmk` from AUR, but this implies that they have to install some things manually first. Also, why does libffi need to be installed manually?

Reply:
The problem I wanted to solve: we do not control the AUR package, @Curry does, but nobody can expect him to keep up with the releases.  
We do have control over the PYPI package.

So I'd rather leave the decision to the users.

**libffi**: It wasn't pulled in as a dependency for any of the packages and the pip installation fails without it as one of the modules requires it.

Reply:
Maybe we could work with @Curry to get direct access and setup a pipeline for releases. It's a direction I'd like to go anyway (for example I plan to auto-update the homebrew update once I figure out how.)

Reply:
I have added `libffi` to the dependencies, so in-case that package isn't installed, it will be installed with the qmk package.

Reply:
I can see that happening in the (near) future, but for now I'm focusing fixing our current docs, so less user stuck following the getting started guide.